documentation/CAMARA-OIDC-profile.md

```diff
@@ -0,0 +1,1020 @@
+
+    * **_invalid_client_**
+
+      Client authentication failed (e.g., unknown client, no client authentication included, or unsupported authentication method). The authorization server MAY return an HTTP 401 (Unauthorized) status code to indicate which HTTP authentication schemes are supported. If the client attempted to authenticate via the `Authorization` request header field, the authorization server MUST respond with an HTTP 401 (Unauthorized) status code and include the `WWW-Authenticate` response header field matching the authentication scheme used by the client.
+
+    * **_invalid_grant_**
+
+      The provided authorization grant `auth_req_id` is invalid or was issued to another client.
+
+    * **_unauthorized_client_**
+
+      The authenticated client is not authorized to use this authorization grant type.
+
+    * **_unsupported_grant_type_**
+
+      The authorization grant type is not supported by the authorization server.
+
+    * **_authorization_pending_**
+
+      The authorization request is still pending as the end-user hasn't yet been authenticated.
+
+    * **_slow_down_**
+
+      A variant of `authorization_pending`, the authorization request is still pending and polling should continue, but the interval MUST be increased by at least 5 seconds for this and all subsequent requests.
+
+    * **_expired_token_**
+
+      The `auth_req_id` has expired. The Client will need to make a new Authentication Request. This error can be obtained when eventually no consent is granted by the user.
+
+    * **_access_denied_**
+
+      The end-user denied the authorization request.
+
+
+* **error_description**
+
+  OPTIONAL. Human-readable ASCII text providing additional information, used to assist the client developer in understanding the error that occurred. Values for the `error_description` parameter MUST NOT include characters outside the set %x20-21 / %x23-5B / %x5D-7E.
+
+
+The parameters are included in the entity-body of the HTTP response using the `application/json` media type.
+
+The following is a non-normative example from a Token Error response:
+
+```htt
+HTTP/1.1 400 Bad Request
+Content-Type: application/json
+```
+```json
+{
+  "error": "authorization_pending"
+}
+```
+
+
+## Client Credentials Flow
+
+The Client Credentials grant type is used to obtain a 2-legged Access Token that does not represent a user. This grant type can only be used when no personal user data is processed, and it is only a valid option to access the CAMARA APIs for these specific scenarios.
+
+
+### Access Token Request
+
+The client makes a request to the token endpoint by adding the following parameters using the `application/x-www-form-urlencoded` format with a character encoding of UTF-8 in the HTTP request entity-body:
+
+* **grant_type**
+
+  REQUIRED. Value MUST be set to `client_credentials`.
+
+
+* **scope**
+
+  OPTIONAL. A space delimited and a case-sensitive list of ASCII strings.
```

Comment:
`scope` must be REQUIRED for CAMARA, because all endpoints will have a scope

Reply:
Changed to REQUIRED.